lib/suspenders/app_builder.rb

```diff
@@ -467,6 +424,10 @@ def configure_rubocop
       template '.rubocop.yml', '.rubocop.yml'
     end
 
+    def run_stairs
+      bundle_command 'exec stairs --use-defaults'
```

Comment:
@taboularasa No, stairs can't run bundle install because it is itself a part of the bundle so `bundle install` is a pre-req to running `rake newb` ... Your question brings up another question: at what point is `suspenders` currently bundling? I'll make sure it happens before stairs...